mvg/plotting.py

```diff
@@ -19,16 +19,19 @@
     (
         [-2, "gray"],
         [-1, "white"],
-        [0, "red"],
-        [1, "blue"],
-        [2, "lime"],
-        [3, "yellow"],
-        [4, "fuchsia"],
-        [5, "green"],
-        [6, "aqua"],
-        [7, "silver"],
-        [8, "fuchsia"],
-        [9, "black"],
+        [0, "blue"],
+        [1, "fuchsia"],
+        [2, "orange"],
+        [3, "aqua"],
+        [4, "pink"],
+        [5, "steelblue"],
+        [6, "olive"],
+        [7, "lavender"],
+        [8, "yellow"],
+        [9, "green"],
```

Comment:
We should remove red, green and yellow colors from the list altogether. For the higher numbers we can choose from other shades of existing colors, e.g., from this [list](https://matplotlib.org/stable/gallery/color/named_colors.html)

Reply:
I had removed those colors in addition to the "lime" color as well since it is bright green, I replace them by beige, mediumseagreen, indigo and lightcoral